api/src/main/java/ca/bc/gov/educ/api/penrequest/model/v1/PenRequestStatusCodeEntity.java

```diff
@@ -15,7 +15,6 @@
 public class PenRequestStatusCodeEntity {
 
   @Id
-  @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
Removed this value, because it was breaking our automation test. The H2 database did not like this throwing a varchar(255) exception when creating this table.